sam-helm-charts/values.yaml

```diff
@@ -0,0 +1,51 @@
+## Namespace
+namespace: default
+cloudProvider: aws
+AWS_SSL_ARN: arn:aws:acm:us-west-1:685980093638:certificate/a4ea3620-d410-456b-ad13-a78fe0de9b9b
+domain: segmentanythingservice.com
+# ====================================================
+# encoder
+# ====================================================
+encoder:
+  enabled: true
+  replicas: 1
+  serviceType: LoadBalancer
+  serviceAnnotations:
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "300"
+  image:
+    name: 685980093638.dkr.ecr.us-east-1.amazonaws.com/sam-service
```

Comment:
@batpad , currently we  don't have a github a action for this deployment, i ma just adding this values  form manual deployment.

Reply:
Yea, I don't think any of these values are sensitive and I think this is fine for now.